extensions/rocksdb-repos/database/OpenRocksDb.cpp

```diff
@@ -118,8 +114,15 @@ rocksdb::DB* OpenRocksDb::get() {
   return impl_.get();
 }
 
-}  // namespace internal
-}  // namespace minifi
-}  // namespace nifi
-}  // namespace apache
-}  // namespace org
+std::optional<uint64_t> OpenRocksDb::getApproximateSizes() const {
+  const rocksdb::SizeApproximationOptions options{ .include_memtabtles = true };
+  const rocksdb::Range range("", "~");
```

Comment:
could keys with leading `~` exist? content paths are prefixed with the content dir's path, I could not find if we turn that into absolute path or not, but if not, could the user set it relative to their home (e.g. `~/content_repository`)?

Reply:
Good point, I suppose it's possible to have a path like this, updated in 2c1c6841b84cd569185323665d9d927a029396e9